lib/services/messaging_services/dms_api_service.dart

```diff
@@ -39,4 +40,62 @@ class DMApiService {
       return null;
     }
   }
+
+  Future<String> roomCreator(String friendId) async {
+    final orgId = _userService.currentOrgId;
+    final String memberID =
+        storageService.getString(StorageKeys.idInOrganization).toString();
+    String endpoint = '/v1/org/$orgId/users/$memberID/room';
+    var body = {
+      "org_id": orgId,
+      "room_member_ids": [memberID, friendId],
+      "room_name": "Prince"
+    };
+    try {
+      final res = await _api.postDM(endpoint, body: body);
+      return res.data['room_id'].toString();
+    } on Exception catch (e) {
+      log.e(e.toString());
+      return 'error';
+    }
+  }
+
+  Future sendDMs(String roomID, String memberID, String message) async {
+    final orgId = _userService.currentOrgId;
+
+    dynamic dmResult;
```

Comment:
Please write comment note on what you'll be doing with this.

Reply:
Done